components/Tabs/src/MDCTabBarViewController.m

```diff
@@ -241,7 +246,7 @@ - (nullable UIViewController *)controllerWithView:(nullable UIView *)view {
 
 - (void)updateLayout {
   CGRect bounds = self.view.bounds;
-  CGFloat tabBarHeight = [[_tabBar class] defaultHeightForItemAppearance:_tabBar.itemAppearance];
+  CGFloat tabBarHeight = [_tabBar sizeThatFits:bounds.size].height;
```

Comment:
We need to set the tab bar's frame to self.bounds before calling sizeThatFits: to ensure its safe area insets are consistent with the view controller's.

Reply:
No, the idea is that depending on where the _tabBar is positioned, this will returned the right size. The first time this is called the height will not be adjusted to account for the safe area inset as the _tabBar currently doesn't have a frame. But every subsequent call will. And since layout happens at least twice, this works as expected.